src/database/DataTypes/IndicatorAttribute.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import Realm from 'realm';
+
+/**
+ * An abstract object which contains metadata describing an indicator row or column.
+ *
+ * @property  {string}            id
+ * @property  {ProgramIndicator}  indicator
+ * @property  {string}            description
+ * @property  {string}            code
+ * @property  {number}            index
+ * @property  {boolean}           isRequired
+ * @property  {string}            valueType
+ * @property  {string}            valueDefault
+ * @property  {string}            axis
+ * @property  {boolean}           isActive
+ */
+export class IndicatorAttribute extends Realm.Object {}
+
+export default IndicatorAttribute;
+
+IndicatorAttribute.schema = {
+  name: 'IndicatorAttribute',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    indicator: 'ProgramIndicator',
+    description: { type: 'string', default: 'placeholderDescription' },
+    code: { type: 'string', default: 'placeholderCode' },
```

Comment:
wondering why defaults for these?

Reply:
`IndicatorAttribute.description`, `IndicatorAttribute.code` are both user displayed fields on tables, so my thought is that having the field blank would potentially be a bit confusing? See for reference, e.g. `Item.code` and `Item.name`.